docs/preview/02-Features/01-General/receive-cloudevents-job.md

```diff
@@ -15,36 +15,75 @@ An Azure Service Bus Topic resource is required to receive CloudEvents on. Cloud
 
 ![Automatically Invalidate Azure Key Vault Secrets](/media/CloudEvents-Job.png)
 
-You can write your own background job(s) by deriving from `CloudEventBackgroundJob` which already takes care of topic subscription creation/deletion on start/stop of the job.
-
 ## Usage
 
-You can easily implement your own job by implementing the `ProcessMessageAsync` method to prcocess new CloudEvents.
+The CloudEvents background job uses the [Arcus Messaging](https://github.com/arcus-azure/arcus.messaging) functionality to receive messages. 
+Make sure you take a look at the [documentation on message handlers](https://messaging.arcus-azure.net/features/message-pumps/service-bus) to fully grasp the possibilities.
```

Comment:
The question is - Should they care how it works under the hood?

Reply:
If you want to handle the `CloudEvent`s from the background job, than you should know about message handlers, right? Bit useless if you only specify the background job without any processing logic?

Reply:
What do you think @fgheysels?

Reply:
I think it's good to have a reference to some background info available.

Reply:
Sounds good to me!

Reply:
Ok, great, I'll consider this PR as done, but we can always change the current docs if necessary.